src/MusicalScore/Graphical/VexFlow/CanvasVexFlowBackend.ts

```diff
@@ -47,8 +47,9 @@ export class CanvasVexFlowBackend extends VexFlowBackend {
         this.canvas.style.zIndex = "0";
         this.inner.appendChild(this.canvas);
         container.appendChild(this.inner);
-        this.renderer = new Vex.Flow.Renderer(this.canvas, this.getVexflowBackendType());
-        this.ctx = <Vex.Flow.CanvasContext>this.renderer.getContext();
+        // @ts-expect-error
+        this.renderer = new VF.Renderer(this.canvas, this.getVexflowBackendType());
```

Comment:
we need `this.canvas as HTMLCanvasElement` here. I just tried npm start and it errors without this change. After this change, npm start works and creates the demo correctly.

Reply:
I know about this one, it is strange. It fails locally but not in the CI.

Reply:
Ah, I see. Still, I see no problem with the casting (`as`), and otherwise npm start won't produce the demo.

Reply:
Done